sdk/core/azure_core_amqp/src/fe2o3/cbs.rs

```diff
@@ -77,8 +78,11 @@ impl<'a> AmqpClaimsBasedSecurityApis for Fe2o3ClaimsBasedSecurity<'a> {
             expires_at
         );
         let cbs_token = CbsToken::new(
-            secret.into(),
-            "jwt",
+            secret,
+            match token_type {
+                Some(token_type) => token_type,
+                None => "jwt".to_string(),
+            },
```

Comment:
```suggestion
            token_type.unwrap_or("jwt".to_string()),
```

I'll only comment on this one for brevity. Please fix other instances.